### ISSUE TITLE
Citation box missing information on web-hosted docpages

### ISSUE THREAD

**Issue by yardasol** [CEMeNT-PSAAP/MCDC#210]
Hi MC/DC! I've been browsing your web-hosted docpages and noticed that the [citation on the website](https://mcdc.readthedocs.io/en/latest/#to-cite-mc-dc) is missing a lot of information:
![Screenshot 2024-07-16 at 12-27-29 MC_DC documentation](https://github.com/user-attachments/assets/b0c18857-d78e-4482-acd8-bced4f71fa5a)

In the [source markdown,](https://github.com/CEMeNT-PSAAP/MCDC/blob/dev/docs/source/index.rst#to-cite-mcdc) all this information is present, so I'm not sure what is causing this discrepancy.

**Comment by ilhamv** [comment]
Hi, @yardasol ! Thanks for pointing that out!
@jpmorgan98 , is it because the website has not been updated with the most recent source yet?

**Comment by jpmorgan98** [comment]
Hmm so this appears to be a docs only problem with importing a colorama version. When I try and build locally I get this numba error:
```
/opt/homebrew/Caskroom/mambaforge/base/envs/mcdc_doc/lib/python3.12/site-packages/numba/core/errors.py:194: UserWarning: Insufficiently recent colorama version found. Numba requires colorama >= 0.3.9
```
My current `colorama` version is 0.4.6. @clemekay any ideas?

**Comment by clemekay** [comment]
Docs throw sphinx.ext.autosummary error starting with commit #6ed8ea8678bc7eb792389cef75bc03a7db3c3c61.

**Comment by clemekay** [comment]
Fixed in #219. Thanks again for pointing this out @yardasol, we hadn't realized that our docs weren't updating!
